karate-core/src/main/java/com/intuit/karate/core/ScriptBridge.java

```diff
@@ -700,19 +727,23 @@ public Object bodyPath(String path) {
     }
 
     public FeatureServer start(String mock) {
-        return start(Collections.singletonMap("mock", mock));
+        return start(Collections.singletonMap("mock", mock == null ? null : Arrays.asList(mock)));
     }
 
     public FeatureServer start(Map<String, Object> config) {
-        String mock = (String) config.get("mock");
-        if (mock == null) {
+        List<String> mocks = (List<String>) config.get("mock");
```

Comment:
so this is an API change ?

Reply:
Its an API change if something calls the start(Map). The one that was used which is start(String) just populates the map.
I can make it support both a Collection or String? (I havent seen any references to it)

Reply:
start(Map) is intended for end-users starting a mock from Java code typically a unit-test / JUnit - so it may need to be retained, should be easy

Reply:
I have fixed that up to support both.